### DISCUSSION TITLE
is persistent volume storage still necessary?

### DISCUSSION THREAD

**Discussion by luizm** [gravitational/teleport#16592] [Q&A] [answered]
Hey folks..

I was taking a look into feature overview: https://goteleport.com/blog/teleport-10-feature-overview, and I saw that teleport from 10.0.2 version onwards, will stop to use persistent volume.

If so, can we update the helm chart to stop created it? https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-cluster

Thank you

**Comment by webvictim** [comment, marked as answer]
This is now targetted to come out in Teleport v11. The Helm chart will be updated once the feature is released.

**Reply by luizm** [reply]
Cool, thank you
